Makefile

```diff
@@ -39,7 +39,8 @@ help:
 # http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-build:
+build: dist
```

Comment:
Good catch.  Not sure if we should just remove the `build` target or make it an alias for `dist`.  Do you find yourself attempting `make build` naturally?  If so, should we document `build` in the help message that is produced when typing `make`?